### DISCUSSION TITLE
How to adjust height of Header in v3.4.2?

### DISCUSSION THREAD

**Discussion by emclab** [react-native-elements/react-native-elements#3254] [Q&A]
Here is the code using `containerStyle` to adjust height of Header. However it didn't work. Also the content stays at the bottom of the header and failed to be centered. The default header height is too big.

         
```
import {  Header } from 'react-native-elements';
          ......
          <Header  
              constainerStyle={{height:30, alignContent:"center", alignItems:"center"}}   //,<==height didn't work. align content/items didn't work as well   
              leftComponent={<HeadLeft/>}
              centerComponent={<HeadCenter />}
              rightComponent={<HeadRight/>}
              
          />
```

The content stays at the bottom of the Header instead of "center". What is the right way to adjust Header height and put the content in center?

**Comment by maximcoding** [comment]
@emclab try to set containerStyle={{height:30}}
 AND  leftContainerStyle={{height: 30}}  OR  rightContainerStyle={{height: 30}}

**Comment by liveakshay** [comment]
Can anyone else answer this question?  The above answer is incomplete.  And yes, the default header is too big in terms of height, so I tried to apply a height="8%" style using ContainerStyle property, which did reduce the size, but all the header components (within the header) are still vertically aligned at the bottom, and there is some unused space above the title etc that seems unnecessary - if we could align the text to top, I can easily reduce the height further using a lower value than 8%.
